README.md

```diff
@@ -137,11 +137,13 @@ filter := bson.D{{"name", "pi"}}
 ctx, cancel = context.WithTimeout(context.Background(), 5*time.Second)
 defer cancel()
 err = collection.FindOne(ctx, filter).Decode(&result)
-if err == mongo.ErrNoDocuments {
+if err != nil {
+  if err == mongo.ErrNoDocuments {
     // Do something when no record was found
     fmt.Println("record does not exist")
-} else {
+  } else {
```

Comment:
For brevity, can you rewrite this slightly as:
```
if err == mongo.ErrNoDocuments {
    // Do something when no record was found
    fmt.Println("record does not exist")
} else if err != nil {
    log.Fatal (err)
}
```

And use four spaces for consistency with indentation in this block?

Reply:
Done!